services/workspace_listing.py

```
@@ -0,0 +1,256 @@
+            projects.append({
+                "id": f"cli:{cp['project_id']}",
+                "name": ws_name,
+                "conversationCount": len(cli_convos),
+                "lastModified": (
+                    datetime.fromtimestamp(last_ms / 1000, tz=timezone.utc).isoformat()
+                    if last_ms
+                    else datetime.now(tz=timezone.utc).isoformat()
+                ),
+                "source": "cli",
+            })
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)